main.js

```diff
@@ -1,16 +1,47 @@
 /* eslint-disable no-unused-vars */
+var Typed = require('typed.js')
+
 var $choiceList = document.querySelector('.choice-list')
 var $submitButton = document.querySelector('#submit')
+var $clearButton = document.querySelector('#clear')
 var $userChoice = document.querySelector('#user-choice')
 var $userName = document.querySelector('#user')
+var $choice = null
+
+var header = new Typed('.header', {
+  strings: ["Let's grab: McDonalds", "Let's grab: Burger King", "Let's grab: Pizza Hut", 'To Be Determined...'],
+  typeSpeed: 50,
+  backSpeed: 50,
+  smartBackSpace: true,
+  showCursor: false
+})
+
+var slogan = new Typed('.slogan', {
+  strings: ["^10000 Can't decide? Let us decide."],
+  typeSpeed: 80,
+  showCursor: false
+})
 
 function renderList() {
-  var $choice = document.createElement('li')
-  $choice.setAttribute('class', 'list')
-  $choice.textContent = $userChoice.value
-  $choiceList.appendChild($choice)
+  if ($userChoice.value !== '' && $userName.value !== '') {
+    $choice = document.createElement('li')
+    $choice.setAttribute('class', 'list')
+    $choice.textContent = $userChoice.value
+    $choiceList.appendChild($choice)
+  }
+}
+
+function clearList() {
+  for (var i = 0; i < $choice.length; i++) {
+    $choiceList.removeChild($choice[i])
```

Comment:
You should declare $choiceList near where it is being used. If you want to define it in a different scope you should at least pass it as an argument to this function rather than relying on it to be in the global scope.